### PR TITLE
Passes reordered + keep (C)initPrime after InlinePass

### PR DESCRIPTION
**Pull request by Kukovec**
Changes the order of passes, so `CoverAnalysisPass` comes before inlining.

### CODE REVIEW

**Comment by Kukovec** [comment]
Once this passes I can merge it into #308 and close that.

File: tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/InlinePassImpl.scala
Comment (konnov) [inline tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/InlinePassImpl.scala:67]: why don't you extend `userOperatorNamesFromOptions` with the operators for `cinit`, `initPrime`, etc.? These names should keep the relevant operators, so later passes would not remove them.

**Review by konnov — APPROVED** [review]
One comment. The rest looks good to me.

**Comment by shonfeder** [comment]
Hi @Kukovec! Just checking in here, as it looks like this PR might be nearly ready for merge, which would help unblock other WIP. LMT if there's anything I can do to help it along :)

**Pull request merged**
